cuda/CMakeLists.txt

```diff
@@ -127,7 +127,7 @@ list(GET CUDA_RUNTIME_LIBS 0 CUDA_FIRST_LIB)
 get_filename_component(GKO_CUDA_LIBDIR "${CUDA_FIRST_LIB}" DIRECTORY)
 
 ginkgo_default_includes(ginkgo_cuda)
-ginkgo_install_library(ginkgo_cuda "${GKO_CUDA_LIBDIR}")
+ginkgo_install_library(ginkgo_cuda "${CUDAToolkit_LIBRARY_DIR}")
```

Comment:
I'm wondering if this is the right approach, given that we now have proper cmake targets for cudart. But I've not looked too much into it.